pipelines/ci-daily.yml

```diff
@@ -0,0 +1,23 @@
+# CI build for developer builds.
+
+variables:
+  Unity2018Version: Unity2018.3.7f1
+  Unity2019Version: Unity2019.2.0f1
+  MRTKVersion: 2.0.0
+  packagingEnabled: false
+
+jobs:
+- job: CIDeveloperValidation
+  timeoutInMinutes: 120
+  pool:
+    name: On-Prem Unity
+    demands:
+    - Unity2018.3.7f1

```

Comment:
Would it make sure to separate Unity2018 and Unity2019 tasks into separate jobs? This way we won't have to be extra careful about having agents with *both* versions present.

Reply:
Yeah, sounds reasonable.

Reply:
I guess we can run tests on 2019 too, wouldn't do any harm.